openml/cli.py

```diff
@@ -43,31 +46,27 @@ def wait_until_valid_input(
     valid input
 
     """
-
     while True:
         response = input(prompt)
         if sanitize:
             response = sanitize(response)
         error_message = check(response)
         if error_message:
-            print(error_message, end="\n\n")
+            pass
         else:
             return response
 
 
 def print_configuration() -> None:
-    file = config.determine_config_file_path()
-    header = f"File '{file}' contains (or defaults to):"
-    print(header)
+    config.determine_config_file_path()
 
-    max_key_length = max(map(len, config.get_config_as_dict()))
-    for field, value in config.get_config_as_dict().items():
-        print(f"{field.ljust(max_key_length)}: {value}")
+    max(map(len, config.get_config_as_dict()))
+    for _field, _value in config.get_config_as_dict().items():
+        pass
 
 
 def verbose_set(field: str, value: str) -> None:
     config.set_field_in_config_file(field, value)
-    print(f"{field} set to '{value}'.")
 
```

Comment:
Ruff removed print, maybe replace this with log or some markers to not be removed by ruff.